tests/test_backend.py

```diff
@@ -3010,6 +3010,26 @@ def test_hashtable_lookup(self):
         self._run_test_case([lookup_results.name], {_INPUT: query})
         os.remove(filnm)
 
-
+    @check_opset_min_version(11)
+    def test_MatrixDiagPart(self):
```

Comment:
nit -- can we rename the test to "test_matrix_diag_part", to make it follow the convention of the tests? Some other tests are also misspelled, but would be good to keep convention.

Reply:
Sure why not.